src/Handler/SentryHandler.php

```diff
@@ -138,7 +138,7 @@ protected function write(array $record): void
             }
 
             $this->client->captureMessage(
-                $record['formatted'],
+                $record['context']['message'],
```

Comment:
If I invoke manual logging - there is no "message" key in the "context" array:

```
        $logger = Injector::inst()->createWithArgs(Logger::class, ['error-log'])
            ->pushHandler(SentryHandler::create('DEBUG')); // Send errors >= DEBUG
        $logger->info('This is a test');
```

`[Notice] Undefined index: message`

It would be good to see exactly how you triggered the error (i.e. is bundle.css being called in a template context or via the `Requirements` API?)

The following might work better:

```
$record['context']['message'] ?? $record['formatted']
```

Reply:
In this case the CSS files were being included via the Requirements API:

```php
    protected function init()
    {
        parent::init();
        Requirements::css('themes/base/build/bundle-index.css');
        Requirements::css('themes/base/build/bundle-print.css', 'print');
    }
```

There is a `$record['message']` property too - which contains the following:
**E_USER_NOTICE: File themes/base/build/bundle-index.css does not exist**

So perhaps?
`$record['context']['message'] ?? $record['message']`